pkg/graveler/graveler.go

```diff
@@ -191,6 +192,18 @@ type Repository struct {
 	StorageNamespace StorageNamespace `db:"storage_namespace"`
 	CreationDate     time.Time        `db:"creation_date"`
 	DefaultBranchID  BranchID         `db:"default_branch"`
+	State            RepositoryState
+	InstanceUID      string
+}
+
+func NewRepository(storageNamespace StorageNamespace, defaultBranchID BranchID) Repository {
+	return Repository{
+		StorageNamespace: storageNamespace,
+		CreationDate:     time.Now().UTC(),
```

Comment:
I don't think UTC matters here.

Reply:
It matters - we should not keep time in localized format since protobuf doesn't save time localization information

Reply:
from `timestamp.pb.go`:
```
func New(t time.Time) *Timestamp {
	return &Timestamp{Seconds: int64(t.Unix()), Nanos: int32(t.Nanosecond())}
}
```

so protobuf's library got your back :)

Reply:
It returns a protobuf timestamp object, we are using the golang variation

Reply:
Right - I thought you were worried about the conversation (I can't think of any other reason to use UTC here)